tests/integration/interface_common_test.py

```diff
@@ -16,11 +16,42 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
+import time
+
+import pytest
 
 import libnmstate
+from libnmstate.error import NmstateVerificationError
 
+from .testlib import assertlib
+from .testlib import cmd as libcmd
 from .testlib import statelib
 from libnmstate.schema import Interface
+from libnmstate.schema import InterfaceIPv4
+from libnmstate.schema import InterfaceIPv6
+from libnmstate.schema import InterfaceState
+
+DUMMY_INTERFACE = 'dummy_test'
+
+
+@pytest.fixture(scope='function')
+def ip_link_dummy():
```

Comment:
- The `ip link` fits a fixture as it is a setup/teardown.
- The `apply` fits the tests in a context manager.

Reply:
Done